src/keri/app/cli/commands/rotate.py

```diff
@@ -19,31 +20,89 @@
 parser.add_argument('--alias', '-a', help='human readable alias for the new identifier prefix', required=True)
 parser.add_argument('--passcode', '-p', help='22 character encryption passcode for keystore (is not saved)',
                     dest="bran", default=None)  # passcode => bran
+parser.add_argument('--file', '-f', help='file path of config options (JSON) for rotation', default="", required=False)
 parser.add_argument('--next-count', '-C', help='Count of pre-rotated keys (signing keys after next rotation).',
                     default=None, type=int, required=False)
 rotating.addRotationArgs(parser)
 
 
+@dataclass
+class RotateOptions:
+    """
+    Configurable options for a rotation performed at the command line.
+    Each of the defaults is depended on by the option merge_args_with_file function.
+    """
+    isith: int | str | list
+    ncount: int
+    nsith: int | str | list = '0'
+    toad: int = 0
+    wits: list = None
+    witsCut: list = None
+    witsAdd: list = None
+    data: list = None
+
+
 def rotate(args):
     """
     Performs a rotation of the identifier of the environment represented by the provided name parameter
 
         args (parseargs):  Command line argument
 
     """
-    data = rotating.loadData(args)
+    opts = merge_args_with_file(args)
     rotDoer = RotateDoer(name=args.name, base=args.base, alias=args.alias,
-                         bran=args.bran, wits=args.witnesses,
-                         cuts=args.cuts, adds=args.witness_add,
-                         isith=args.isith, nsith=args.nsith,
-                         count=args.next_count, toad=args.toad,
-                         data=data)
+                         bran=args.bran, wits=opts.wits,
+                         cuts=opts.witsCut, adds=opts.witsAdd,
+                         isith=opts.isith, nsith=opts.nsith,
+                         count=opts.ncount, toad=opts.toad,
+                         data=opts.data)
 
     doers = [rotDoer]
 
     return doers
 
 
+def empty_options():
+    """
+    Empty rotation options used for merging file and command line options
+    """
+    return RotateOptions(
+        isith=None, ncount=None
+    )
+
+
+def merge_args_with_file(args):
```

Comment:
Convention in KERIPY is to use camelcase for method names, please fix

(I won't comment on all methods, but they all need to be fixed)

Reply:
I updated the method names to all be camelCase.